scripts/verify-pr.py

```
@@ -0,0 +1,608 @@
+    if not api_key:
+        logger.error("ANTHROPIC_API_KEY environment variable not set")
+        raise ValueError(
+            "ANTHROPIC_API_KEY environment variable not set. "
+            "Please set it before running this script."
+        )
+    logger.debug("API key validation successful")
+
+
+def main(
+    pr_identifier: str = typer.Argument(..., help="PR number, branch name, or comparison"),
+    json_output: bool = typer.Option(False, "--json", "-j", help="Output results in JSON format"),
```

Ruff (0.14.4)

Boolean-typed positional argument in function definition

(FBT001)